cmd/incusd/acme.go

```diff
@@ -78,7 +78,7 @@ func autoRenewCertificate(ctx context.Context, d *Daemon, force bool) error {
 
 	domain, email, caURL, agreeToS, challengeType := s.GlobalConfig.ACME()
 
-	if domain == "" || email == "" || !agreeToS || challengeType == "" {
+	if domain == "" || email == "" || !agreeToS {
```

Comment:
We should still keep this one. challengeType is still returned and it has a default value so it's fine to ensure it's not empty.